gems/pending/metadata/linux/LinuxUtils.rb

```diff
@@ -141,6 +141,44 @@ def self.parse_systemctl_list(lines)
       end.compact!
     end
 
+    def self.collect_interface(interfaces, interface)
+      mac_addr = interface[:mac_address]
+      return if mac_addr.blank?
+
+      existing_interface = interfaces[mac_addr]
+      if existing_interface.blank?
+        interfaces[mac_addr] = interface
+      else
+        interface[:name] += "," + existing_interface[:name]
+        existing_interface.merge!(interface)
+      end
+    end
+
+    def self.parse_network_interface_list(lines)
```

Comment:
hm I though I written tests for it, there has to be some test stealing monster :-)